src/wasm/test-fixtures/FIXTURES.md

```
@@ -13,6 +15,55 @@ All three are v2.x format files using the Propellerhead `CAT `/`RB40HEAD` chunk
 container. They were chosen because they differ in title/author/info content and
 in the sub-format flags observed in the `HEAD`/`GLOB` chunks.
 
+## v1 / v1.5 MIDI-container fixtures (`MThd`)
+
+ReBirth 1.0 / 1.5 song files are **not** the v2 IFF container. Official
+Propellerhead SongPacks from 1997 are Standard MIDI Type 1 files whose first
+track is a copyright meta-event:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)